docs/features/dashboard-liabilities-separation.md

```
@@ -0,0 +1,112 @@
+- `backend/src/main/java/com/picsou/service/DashboardService.java` —
+  `buildDistribution(accounts, divisor, …)` with per-side divisors
+- `frontend/src/pages/dashboard/DashboardPage.tsx` — mode-dependent chart
+  title, renders `LiabilitiesCard`
+- `frontend/src/components/shared/NetWorthChart.tsx` — tooltip reads row `pnl`
+- `frontend/src/components/shared/LiabilitiesCard.tsx` — liabilities list
+- `frontend/src/demo/data/dashboard.ts` — demo mock ships one loan so demo
+  mode exercises the card
+
+### Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)